integration_tests/CMakeLists.txt

```diff
@@ -293,7 +293,8 @@ RUN(NAME test_dict_06        LABELS cpython llvm c)
 RUN(NAME test_dict_07        LABELS cpython llvm)
 RUN(NAME test_dict_08        LABELS cpython llvm c)
 RUN(NAME test_dict_09        LABELS cpython llvm c)
-RUN(NAME test_dict_10        LABELS cpython llvm) # TODO: Add support of dict with string in C backend 
+RUN(NAME test_dict_10        LABELS cpython llvm) # TODO: Add support of dict with string in C backend
+RUN(NAME test_dict_11        LABELS cpython c) # TODO: Add LLVM support
```

Comment:
We can't move ahead without enabling LLVM backend for any test. Please make it work with LLVM before C.

Reply:
That's a good point, the LLVM backend is a "lowering" backend, so it ensures our design is good. The C/C++/Julia backends are high level backends, so they don't reveal all issues. WASM is also a lowering backend.

Reply:
Yep, that sounds right. Thanks, I fixed it.